src/main/java/io/confluent/connect/jdbc/dialect/PostgreSqlDatabaseDialect.java

```diff
@@ -280,6 +312,13 @@ protected ColumnConverter columnConverterFor(
         if (UUID.class.getName().equals(columnDefn.classNameForType())) {
           return rs -> rs.getString(col);
         }
+
+        if (OffsetDateTime.class.getName().equals(columnDefn.classNameForType())) {
+          return rs -> rs.getObject(col, OffsetDateTime.class).format(ISO_OFFSET_DATE_TIME);
```

Comment:
Using `format` here with explicit formatters is strictly not necessary as you could use `toString` instead (the value output is the same), but the usage of an explicit formatter is better as there is no risk that it would change in the future (where as `.toString` representations can)